kyverno-cli/Dockerfile

```
@@ -0,0 +1,58 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
+
+LABEL version="1.0.0"
+LABEL repository="http://github.com/redhat-cop/github-actions"
+LABEL homepage="http://github.com/redhat-cop/github-actions/kyverno-cli"
+LABEL maintainer="Red Hat CoP"
+LABEL "com.github.actions.name"="kyverno-cli"
+LABEL "com.github.actions.description"="Run kyverno using BATS"
+LABEL "com.github.actions.branding.icon"="monitor"
+LABEL "com.github.actions.branding.color"="purple"
+
+RUN microdnf install --assumeyes --nodocs tar wget python3 git findutils diffutils bc && \
```

Might also want to update with any package releases since base image was published
```suggestion
RUN microdnf upgrade --assumeyes && \
    microdnf install --assumeyes --nodocs tar wget python3 git findutils diffutils bc && \
```